Styles/Classes/ViewStyle.swift

```diff
@@ -56,6 +56,23 @@ public final class ViewStyle: NSObject {
             }
         }
 
+        var isLayout: Bool {
+            switch self {
+            case .backgroundColor:
```

Comment:
```swift 
case .roundCorners: return true
default: return false ```

Reply:
I rather have switch without default if possible. Default case is more likely to be forgotten.